apps/mail/components/context/command-palette-context.tsx

```
@@ -8,92 +29,707 @@ import {
 type CommandPaletteContext = {
   open: boolean;
   setOpen: (open: boolean) => void;
   openModal: () => void;
 };
 
-type Props = {
-  children?: React.ReactNode | React.ReactNode[];
-};
-
-type CommandItem = {
+interface CommandItem {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'CommandItem'. Consider to delete it or rename it.

'CommandItem' is defined here:


(lint/suspicious/noRedeclare)